plugins/fastem_roa_import_export.py

```diff
@@ -0,0 +1,341 @@
+# -*- coding: utf-8 -*-
+"""
+Created on 13 December 2023
+
+@author: Nandish Patel
+
+Gives ability to use Import Export ROAs tool under Help > Development.
+
+Copyright © 2023 Nandish Patel, Delmic
+
+This file is part of Odemis.
+
+Odemis is free software: you can redistribute it and/or modify it under the terms of the GNU
+General Public License version 2 as published by the Free Software Foundation.
+
+Odemis is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even
+the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General
+Public License for more details.
+
+You should have received a copy of the GNU General Public License along with Odemis. If not,
+see http://www.gnu.org/licenses/.
+"""
+
+import csv
+from enum import Enum
+import logging
+import wx
+from wx.lib.embeddedimage import PyEmbeddedImage
+
+from odemis.gui.plugin import Plugin
+
+
+IMPORT_IMAGE = PyEmbeddedImage(
```

Comment:
The "source" of these images (ie, png) should be part of this PR too. You can add them to the `images` folder.